serverless/sql-databases/concepts.mdx

```diff
@@ -23,7 +23,80 @@ categories:
 </Concept>
 
 <Concept opened>
- ## Credentials
+## Backup
 
- Credentials let you connect to your database via a secret key.
-</Concept>
\ No newline at end of file
+A database backup is a copy of the operational state, architecture and stored data of a database. It allows to restore a database if it crashes, is corrupted or is lost.
+</Concept>
+
+<Concept opened>
+## Cold start
+
+Cold start is the time an idle database takes to handle the first request. It usually takes less than five seconds.
+</Concept>
+
+<Concept>
+## Connection pooling
+
+Connection pooling is a cache of database connections maintained so that the connections can be reused when future requests to the database are required. It enhances performance of executing commands on a database. 
+</Concept>
+
+<Concept>
+## Credentials
+
+Credentials let you connect to your database via a secret key.
+</Concept>
+
+<Concept>
+## Engine
+
+A database engine is the software that stores and retrieves your data from a database. Serverless SQL databases use the PostgreSQL engine.
+</Concept>
+
+<Concept>
+## High availability
+
+Databases are replicated on a copy of themselves that is not accessible by the user. This "standby" database guarantees that the service remains available in case of failure.
```

Comment:
The suggested definition corresponds more to "Read replicas". It's a way of providing High Availability, but we currently don't plan to use this feature to provide High Availability (ie. mainly because this doubles-up costs, and we might be able to juste restart a database if one crashes on the VM it's currently hosted on).
We will  provide more concrete information to the users when HA is defined and proven. However this is a complicated topic related to SLAs, so I suggest we keep these information for now.
```suggestion
Databases can become unavailable due to several issues (host crashes, disk corruption, network unavailability...) High Availability means being able to ensure database stays available most of the time and recovers fast from issues so that applications relying on it stay available.
```